src/config.ts

```diff
@@ -9,6 +9,7 @@ export class Config {
 	TWILIO_SENDGRID_API_KEY = '';
 	AWS_SES_REGION = '';
 	AWS_SMS_REGION = '';
+	FIREBASE_CERT_PATH = '';
```

Comment:
Can you fix this to just take the cert value instead of a path?

Reply:
The content is an object/json file? Should I put all the content using JSON.stringify() ?

Reply:
Can we just take it as a string value and pass it to the provider as a string?

Reply:
Yes, we can pass the entire file as string and we need to use JSON.parse() in provider constructor/initialize

Reply:
Ok. Lets do that

Reply:
Example of the json file content that we need in the .env

I don't know if is good idea pass all content in a single environment key.

```
{"type":"service_account","project_id":"airhorn-demo","private_key_id":"ffcc76c6d2314bf2a5efd82d342ff72cb3fce68a","private_key":"-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCYceTShZ/WU3Om\ntglOsXj7pcdyKP/1Gwsb+DQesBICbZEQp71R8CJ5JuXCBdJOQMBBbymvmzvVz3Ts\nGAzJ30Socj/fbxSI8FGO4chkSRdci6mIDGo4SzRe2D4cghRMZ+BtWD3X0XWe5Bki\nlSE/gtLUcTukcDumbUx1g/3VQsROoADnjjLBcFpmrodwPQBJXR3WBjoltxSyMkgd\nerUhCNUlDYc/6Ydo2/zhOoy55/UwCZ4aXQAtT3MkIbWZLGS4DKM+AYqRWS1IOrcM\n+FkHdNhop5aY7MSonkubkLIdXCtVQ3EF52jNeswZOLr0I/mDvCakqogHVqPAfcOB\nJ7SlVcbZAgMBAAECggEABTIs8kMuoHm8dB2ZtF0T97bYim13k56x+p3I2qrAww9X\nPcCL5p8HMRwzF1wxr9a2Pty3kzuDgaKGNJW5b4g3cRWLfrWswbdNk6+iMA82N20w\nEHnaL9REuGle4kOe8RTsJz9LWf/IO6vVvVIk3Zhw0B/Saw94wa/RcU14DB9wF70V\nV6aWXqwCz032GDnvIO6NL+702fIJcMn/AXahYObDxYsQxmPif4ZMu+quXfEjdusE\nY6sdQzewGoKuGU9ljPpoYbOZTjwPGZKLuRja27PwbztWGUuFzj3TR25j8GyKkuRP\ncA7ZjdLTRmA96y7yS0s9ptVho8gFbRUX6tgh0t7ocQKBgQDQbVMlzihkO2qxLY6k\nHi43iliHjV7PaGJvIBp0UsgF7J+Pqoon7Ms5HCMXK+HXj+9HSqJyMLEfgWkUyOIr\nklOXdpl2XYVpLIzIsBkNZz6WCD7HTT1t/7DBMe9gV3uefvB2xEoG8G7T1WUulJD6\nnNdxoikWDFTOyg6c1a8qYIaMKQKBgQC7PXS6JfkrS2KLCQl+8TnNSjmHYbwbpmPX\n2S3daYsa6MegewAQf/iI4I27E4WW2GsPoFNULKIl8Ya4YBMdbrX4+epae6HWSU2y\ngR29gZuAGEOj78hxuA4K7hK3Ftn3KdD/qtSmtyaYH4Ontv0xTzUDFZfFpqQBWZof\nYDHVn/u7MQKBgDYpEoAycCCLqA/ebMCM7ymTk6m1vLtcvqdtyjIMCNoxQM1OXZQ+\nXhi7DQtKZedkCaOW61uM95neQPYbtVnpLjufoAU4DCDmaLcFu5zWBVdrURTtJ1dc\nKAwr864vTXpENixGmY3wEHyPaTmPdZUKCJoIr3CBR9BshRuQE3KJnqzxAoGAbz/N\nigij6lhpHYjqDnWPTW1rjwe/94G+UvJkMOjT0NI8Qhqy6BrQ9arOKzgnKIA/5FHz\nENFQKlE+JXBYCyXLkzoSR16rwqFdO9FBdWZ7mPlRvybqJr8gANiBYC2Apj0/Q5kh\nrkil8r7Ov6iW+WRBcCq1OaeQTIkLbNb/5RigPhECgYBdXF0LA1KMH2yI8/2QUpCJ\noEmQIaNPAjrEI4c1ScWf63RcA7J1a/audyNdD3snOdbLkk6O7HG/eZaZc1Ehdocl\nyH9JUM1Bm2o+5aw1mIrpnPYTGTngY7Al6PCCaLQLEo1dHVY2ViiatpELU7EjyLmq\n6NRxbOKjQrMgo7JmBdbnRw==\n-----END PRIVATE KEY-----\n","client_email":"firebase-adminsdk-5y5d1@airhorn-demo.iam.gserviceaccount.com","client_id":"105896876194716553261","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_x509_cert_url":"https://www.googleapis.com/robot/v1/metadata/x509/firebase-adminsdk-5y5d1%40airhorn-demo.iam.gserviceaccount.com"}
```

Reply:
I like this approach better as this is a library that others use. This means that they control the file system and can decide if they want to have the cert as a file or pull it from a secrets repository